person_capture/gui_app.py

```
@@ -8281,6 +8555,122 @@ def _quantile_lut(src_ch: np.ndarray, dst_ch: np.ndarray, mask: np.ndarray) -> O
+
+            changed, repaired_tmp = self._repair_wic_with_yuv444_color_match(ref_png, clean_png)
+            if changed <= 0 or not repaired_tmp:
+                return False, "color_match_noop", 0
+            valid, invalid_why = self._validate_hdr_sdr_export_image(repaired_tmp, None)
+            if not valid:
+                return False, f"color_match_invalid:{invalid_why}", 0
+            try:
+                if os.path.exists(out_path):
+                    os.remove(out_path)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)